### PR TITLE
En 5253 deploy changes

### PR DESCRIPTION
**Pull request by bogdan-rosianu**

### CODE REVIEW

File: cmd/filegen/main.go [hunk 1]
```diff
@@ -394,50 +391,13 @@ func generateFiles(ctx *cli.Context) error {
...
-	initialNodesKeyGenerator = signing.NewKeyGenerator(suite)
-	numObservers := numOfShards*numOfObserversPerShard + numOfMetachainObservers
-	for i := 0; i < totalNumOfNodes+numObservers; i++ {
-		pkHex, skHex, err = getIdentifierAndPrivateKey(initialNodesKeyGenerator)
-		if err != nil {
-			return err
-		}
 
-		addrPkHex, _, err = getIdentifierAndPrivateKey(balancesKeyGenerator)
+		pkHexForNode, skHexForNode, err := getIdentifierAndPrivateKey(getNodesKeyGen(consensusType))
```
Comment (SebastianMarian) [inline cmd/filegen/main.go]: You should extract in variable the return of getNodesKeyGen(consensusType), as this method could return nil, and in this case you would get panic in first line of getIdentifierAndPrivateKey method

File: cmd/filegen/main.go [hunk 2]
```diff
@@ -451,6 +411,24 @@ func generateFiles(ctx *cli.Context) error {
 		if err != nil {
 			return err
 		}
+
+		numObservers := numOfShards*numOfObserversPerShard + numOfMetachainObservers
```
Comment (SebastianMarian) [inline cmd/filegen/main.go]: You can extract this line together with the calculation below -> totalAddressesWithBalances-numObservers in one variable outside of this for and then you can use it directly in next line -> if i < precalculatedVariable. Actually you don't need to recalculate this in each iteration as this value would not be changed.

Comment (bogdan-rosianu) [reply cmd/filegen/main.go]: done

File: cmd/filegen/main.go (hunk 1 above)
Comment (bogdan-rosianu) [reply cmd/filegen/main.go]: done

**Pull request merged**